src/bridge/cockpitpeer.c

```diff
@@ -400,7 +401,22 @@ on_other_control (CockpitTransport *transport,
           cockpit_transport_send (transport, NULL, reply);
           g_bytes_unref (reply);
         }
+    }
 
+  /* cockpit-bridge --privileged expects to receive a copy of our stderr */
+  else if (g_str_equal (command, "send-stderr") &&
+           cockpit_json_get_bool (self->config, "privileged", FALSE, &privileged) && privileged)
```

Comment:
Why the extra check for `"privileged"` here?

Reply:
I think we covered this one before: it would be weird to get this message from something that's not a privileged bridge, and it might cause us to engage in weird behaviour (like trying to send an fd to ssh).  Better to treat that as unexpected than to attempt to fulfill the request.  We can expand it later if we want...